core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
```

Comment:
Yes, you do; the function ends up as a method of the State object in exp_domain. Any tests move accordingly.